app/controllers/Application.scala

```diff
@@ -82,36 +83,39 @@ class Application(
   }
 
   def supportLanding(): Action[AnyContent] = CachedAction() { implicit request =>
+    implicit val settings: Settings = settingsProvider.settings()
```

Comment:
this is so much clearer than an ActionBuilder

Reply:
I remember you suggested at one point making it a trait with `def` and mixing it into the controller, to avoid this repetition. Though I'm not convinced the repetition is necessarily bad (it at least demonstrates to someone reading the controller that fetching the settings and having a settings surrogate key go together). Did you have any more thoughts on that?

Reply:
Interesting question. I'm not sure which is preferable to be honest. The current approach or mixing in something like:
```
trait SettingsImplicits {
  def settingsProvider: SettingsProvider
  implicit def settings: Settings = settingsProvider.settings()
}
```
to each controller. The current approach is more explicit - since a settings instance is getting created in the action, it's easier to see which actions require the result to be returned with a Surrogate-Key header. Conversely, it is more verbose.

Reply:
let's go anti-Scala and favour verbicitness (my new word for verbosity + explicitness)

Reply:
Cool. It's a simple change if people get fed up with the verbosity in future.